### PR TITLE
Created V-0.1

### PR DESCRIPTION
**Pull request by MykolaFatkullin**
- Created model Movie;
- Created MovieDao;
- Created MovieService;
- Created HibernateUtil;
- Created Injector;
- Created annotations: Dao, Inject;

### CODE REVIEW

File: src/main/java/com/tickets/dao/impl/MovieDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    @Override
+    public Movie add(Movie movie) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            Long movieId = (Long) session.save(movie);
+            transaction.commit();
+            movie.setId(movieId);
```
Comment (Max-Pochepets) [inline src/main/java/com/tickets/dao/impl/MovieDaoImpl.java]: check [this](https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_01.html) one more time

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]
make sure all your files have empty line in the end

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]
what about [this](https://github.com/NikolayFatkullin/TicketsShop/pull/1#discussion_r564619897) one?

**Review by Max-Pochepets — APPROVED** [review]
👍

File: src/main/java/com/tickets/dao/impl/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert Movie entity", e);
```
Comment (Sofasmile) [inline src/main/java/com/tickets/dao/impl/MovieDaoImpl.java]: let's create your own exception

**Review by Sofasmile — CHANGES_REQUESTED** [review]

File: src/main/java/com/tickets/dao/impl/MovieDaoImpl.java [hunk 3]
```diff
@@ -25,7 +26,7 @@ public Movie add(Movie movie) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new RuntimeException("Can't insert Movie entity", e);
+            throw new DataProcessingException("Can't insert Movie entity", e);
```
Comment (Sofasmile) [inline src/main/java/com/tickets/dao/impl/MovieDaoImpl.java]:
```suggestion
            throw new DataProcessingException("Can't insert Movie entity " + movie, e);
```

**Review by Sofasmile — CHANGES_REQUESTED** [review]

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**